packages/react/src/components/sectional-banner/sectional-banner.tsx

```diff
@@ -61,9 +61,10 @@ function abstractContainer(
     return styled.section<AbstractContainerProps>`
         background-color: ${bgColor};
         border: 1px solid ${(props) => (color ? props.theme.notifications[color] : props.theme.main['primary-3'])};
+        border-radius: var(--border-radius-2x);
         box-sizing: border-box;
         line-height: ${({ isMobile }) => getLineHeight(isMobile)}px;
-        padding: ${(props) => (props.isMobile ? 'var(--spacing-3x) var(--spacing-2x)' : 'var(--spacing-2x)')};
+        padding: ${(props) => (props.isMobile ? 'var(--spacing-3x) var(--spacing-2x)' : 'var(--spacing-2x) var(--spacing-3x)')};
```

Comment:
The story prescribed a padding-bottom of 20px. Given the 4px space above the title and icon, that would make the spacing even between the top and the bottom.

Reply:
done, all styles aren't based on the story anymore but on the figma design